### ISSUE TITLE
it worked for me before but from today i think is not working anymore why ?

### ISSUE THREAD

**Issue by zeldoxgamer** [m1dugh/RosettaStonks#45]
**Describe the bug**
it worked for me before but from today i think is not working anymore why ?

**Browser**
- [ ] firefox
- [x] chrome

**version**: 

**rosettastone product**
- [x] Foundations
- [x] Fluency builder

**To Reproduce**

**Expected behavior**

**Comment by m1dugh** [comment]
You ticked both products, did you try both, or only fluency builder ?

**Comment by zeldoxgamer** [comment]
i try in the both , before its working in the same time in rosetta foundations , and after maybe 10min in fluency builder , now its not working in the both of him i think the glitch are using its fixed from the company

**Comment by zeldoxgamer** [comment]
or maybe its need an update

**Comment by moataz2558** [comment]
it worked for me, but i waited a whole day for it to appear

**Comment by prettypasta** [comment]
> it worked for me, but i waited a whole day for it to appear

hello please moataz how did it worj for you i didnt know how to do it because i have a problem with my computer , can we talk on instagram or another app please ? can i give you my password and you try for me please

**Comment by Ryuzaki001** [comment]
Why isn't it working now? It was working in the morning, but now it isn't.

**Comment by zeldoxgamer** [comment]
i found the solution. now its taking more time the show the final result you just need to wait and dont spam to avoid a big hours

**Comment by moondust011** [comment]
how long did you wait ?

**Comment by zeldoxgamer** [comment]
i dont know i checked after maybe 2 hours and i found it worked

**Comment by aminezairig** [comment]
yes thats it you just have to wait for it to appear dont spam i think the servers are full thats why takes long for it to appear

**Comment by zeldoxgamer** [comment]
hta ana bhalk makaydkhlonich les cours ms kandwz sway3 wakha maydkhlonich w kantsna hta tdoz chi 1h wla 2h wghan checki

**Comment by aminezairig** [comment]
then lets check in later ou whoever got the time added to their accounts up vote or like just so we know if it works at the same time for everyone

**Comment by BXRl** [comment]
> i dont know i checked after maybe 2 hours and i found it worked

more than 5 hours and nothing changed wich it annoying idk whats the reason for time not added ill wait for 24 hours and ill update yall

**Comment by Salahajouaou** [comment]
The issue is clearly due to the excessive pressure on the servers

**Comment by Ryuzaki001** [comment]
it works tnx

**Comment by Ryuzaki001** [comment]
more than an 12h

‫في الثلاثاء، 3 ديسمبر 2024 في 6:27 م تمت كتابة ما يلي بواسطة
‪zeldoxgamer‬‏ ***@***.***‬‏>:‬
